FolfApp/src/components/courseDetails.re

```diff
@@ -0,0 +1,57 @@
+module CourseDetails = {
+  include ReactNative;
+  include ReactRe.Component.JsProps;
+  let name = "CourseDetails";
+  type par = Js.t {. red : Js.Null.t int, white : Js.Null.t int, blue : Js.Null.t int};
```

Comment:
We could possibly do that with some clever types, but we'd need to restructure the data. I'll save it for later